spec/factories/parser_templates.rb

```diff
@@ -10,8 +10,7 @@
 
 FactoryBot.define do
   factory :parser_template do
-    name 'Copyright'
+    name      { Faker::Name.first_name }
```

Comment:
will this work without the wrapping squigglies => `name Faker::Name.first_name`

Reply:
unfortunately not : https://devhints.io/factory_bot

Reply:
The wrapping squiggles means that you don't get the same value out of the block if the same factory is used multiple times in the same test.